pages/services/edge-lb/1.0.0/pool-configuration/index.md

```diff
@@ -16,489 +16,8 @@ The configuration reference containing all possible options and descriptions of
 dcos edgelb show --reference
 ```
 
-# Edge-LB v1.0.0
+# API Versions
 
-The tables below describe all possible configuration options. The majority of fields have sensible defaults and should be modified with caution.
+A new top level configuration field named `apiVersion` was introduced in Edge-LB v1.0.0. The two models are almost identical with one important difference: `pool.haproxy.backends.servers` (apiVersion `V1`) has been replaced with `pool.haproxy.backends.services` with a more intuitive way to select services / backends for HAProxy.
```

Comment:
Can we augment this text with:
"NOTE: For backwards compatibility the `apiVersion` field defaults to `v1` . Hence, in order to use the `v2` config a user needs to explicitly set the `apiVersion` to `v2`.